lib/greed.rb

```diff
@@ -0,0 +1,12 @@
+absolute_path = File.expand_path(File.dirname(__FILE__))
+require absolute_path + '/greed/game'
```

Comment:
Doing path manipulation like this is a little awkward (as you may have felt).

It would be better just to assume all of `lib` is in the load path and require things relative to that.

```
require 'greed/game'
```

If you need to add something to your load path you can do that in ruby's invocation


```
ruby -Ilib test/whatever.rb
```